### ISSUE TITLE
[Firebase/Crashlytics] Failed to download settings. Error 404.

### ISSUE THREAD

**Issue by lolgear** [firebase/firebase-ios-sdk#6857]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 12.1
  * Firebase SDK version: 7.0.0
  * Installation method: `Swift Package Manager` (`7.0.0-spm-beta` branch)
  * Firebase Component: `Crashlytics`

### [REQUIRED] Step 2: Describe the problem
After following steps in your guide, I have encountered an error 404 "Failed to download settings".

I also try to inspect a `NSHTTPURLResponse` instance.

```objective-c
<NSHTTPURLResponse: 0x28160d640> { URL: https://firebase-settings.crashlytics.com/spi/v2/platforms/ios/gmp/GOOGLE_CLIENT_ID/settings?build_version=29&display_version=0.1.0&instance=<SOME_ID>&source=1 } { Status Code: 404, Headers { ... }
```

#### Steps to reproduce:

1. I created fresh app on Firebase. ( No migration from Fabric )
2. I downloaded GoogleService-Info.plist and added it to a project.
3. I installed Firebase via SPM ( branch `7.0.0-spm-beta` ).
4. I added `Firebase.configure()`.
5. I added **ONLY** `FirebaseCrashlytics` to a target.

**Comment by ryanwilson** [comment]
Hey @lolgear,

You may need to enable Crashlytics in the Firebase Console first. We should definitely have a better error case but please check that and hopefully that unblocks you!

**Comment by lolgear** [comment]
@ryanwilson 
Yes, you are right :)
Thanks a lot for help!

One note:
It would be nice to have Crashlytics.crash() method. It was specified in documentation, but it doesn't exist in 7.0.0 build. I really don't know how to crash app without "known" bugs in this app :)

**Comment by ryanwilson** [comment]
You're welcome!

Which docs are you looking at? I believe those are docs for the old SDK, you can find the current documentation here: https://firebase.google.com/docs/reference/swift/firebasecrashlytics/api/reference/Classes/Crashlytics. 

In terms of a replacement for the `crash()` method, cc @samedson who can provide guidance.

I'll close this issue since the original issue is solved, but we can continue the conversation here.

**Comment by samedson** [comment]
Here are some recommendations for forcing a crash: https://firebase.google.com/docs/crashlytics/test-implementation?authuser=0

**Comment by lolgear** [comment]
@samedson I forgot about `fatalError()` :(
